jobs/rootfs-builder.jpl

```diff
@@ -81,6 +96,11 @@ node("debos && docker") {
     def arch = "${params.ROOTFS_ARCH}"
     def pipeline_version =  "${params.PIPELINE_VERSION}"
     def docker_image = "${params.DOCKER_BASE}debos"
```

Comment:
I'd rather have docker_image variable value set in a single place based on `rootfs_type`.
Maybe using `switch..case` based on `rootfs_type` could be a good solution?